File: cpp/isPrime/isPrime.cpp

#include <iostream>
uint32_t primes =1;

bool isPrime(uint32_t number){
   uint32_t i;
   for (i = 2; i < number; i++ ){
       if (number % i == 0){
           return false;
       }
   }
   return true;
}

int main(){
    uint32_t k;
    for (k = 3; k < 200000; k++){
        if (isPrime(k)){
           primes++;
        }
     }
     std::cout << primes << std::endl;
}

File: src/lua/isPrime/isPrime.lua
primes = 0


function isPrime(number)
    for i = 2, number-1, 1 do
        if (number % i) == 0 then
           return false
        end
    end
    return true
end


for v = 3, 2000000, 1 do
    if (isPrime(v) == true) then
        primes = primes + 1
    end
end
print(primes)
